src/server/scripts/Northrend/Naxxramas/boss_faerlina.cpp

```diff
@@ -58,38 +72,48 @@ class boss_faerlina : public CreatureScript
         struct boss_faerlinaAI : public BossAI
         {
             boss_faerlinaAI(Creature* creature) : BossAI(creature, BOSS_FAERLINA),
-                _frenzyDispels(0), _introDone(false), _delayFrenzy(false)
+                _frenzyDispels(0)
+            {
+            }
+
+            void SummonAdds()
             {
+                me->SummonCreatureGroup(SUMMON_GROUP_WORSHIPPERS);
+                if (Is25ManRaid())
+                    me->SummonCreatureGroup(SUMMON_GROUP_FOLLOWERS);
             }
 
+            void InitializeAI() override
```

Comment:
Reset() is called right after evade mode starts. We don't want adds to resummon at this time (it allows them to be pulled without engaging the boss, as the boss is still busy running back). InitializeAI() handles initial spawn, while JustReachedHome() handles respawn on reset.